### PR TITLE
DO NOT MERGE - added sparkswap to ccxt

### PR DESCRIPTION
**Pull request by dannypaz**
This is an example PR so that we can view the changes to this branch. All edits to CCXT should be made against the `adding-sparkswap` branch.

The only file we need to add is `sparkswap.js`. The CCXT build command will auto generate all documentation and php/python files which are not required to be checked in to the library initially.

PLEASE see the wiki for steps on how to build CCXT

### CODE REVIEW

File: js/sparkswap.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            },
+            'api': {
+                'public': {
+                    'get': [],
+                },
+                'private': {
+                    'get': [
+                        'v1/admin/healthcheck',
+                        'v1/markets', // get supported markets
+                        'v1/markets_stats', // get market stats for a specific market
```
Comment (martineehrlich) [inline js/sparkswap.js]: v1/market_stats

**Review by martineehrlich — CHANGES_REQUESTED** [review]

**Review by martineehrlich — APPROVED** [review]
LGTM

File: js/sparkswap.js [hunk 2]
```diff
@@ -0,0 +1,544 @@
...
+                // We do not support CORS w/ headers
+                'CORS': false,
+                // Sparkswap exchange has both private and public APIs
+                'private': false,
+                'public': false,
+                // Methods that we support for sparkswap
+                'cancelOrder': 'emulated',
+                'createDepositAddress': 'emulated',
+                'createOrder': 'emulated',
+                'deposit': 'emulated',
```
Comment (martineehrlich) [inline js/sparkswap.js]: We did not implement this. Were we supposed to?

**Comment by dannypaz** [comment]
@martineehrlich @treygriffith closing this PR but _not_ deleting the branch. The code in ccxt has been heavily modified and no longer fully represents the changes. https://github.com/ccxt/ccxt/pull/3974

**Pull request closed without merging**